selftests/pre_release/jobs/features.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import time
+
+from avocado import Test
+from avocado.core import exit_codes
+from avocado.core.job import Job
+
+BOOLEAN_ENABLED = 'on'
+BOOLEAN_DISABLED = 'off'
+
+
+class JobAPIFeaturesTest(Test):
+
+    def check_exit_code(self, exit_code):
+        """Check if job ended with success"""
+        expected_exit_code = self.params.get('exit_code',
+                                             default=exit_codes.AVOCADO_ALL_OK)
+        self.assertEqual(expected_exit_code, exit_code)
+
+    def check_file_exists(self, file_path, assert_func):
+        """Check if a file exist or not depending on the `assert_func`"""
+        assert_func(os.path.exists(file_path))
+
+    def check_file_content(self, file_path, assert_func, content):
+        """
+        Check if `content` exist or not in a file depending on the
```

Comment:
Docstring one line or multiple lines following PEP257.

Reply:
As this is a POC yet, I'm not worried about docstring here.